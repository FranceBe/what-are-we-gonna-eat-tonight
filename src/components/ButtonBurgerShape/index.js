import ButtonBurgerShapeComponent,
{ ButtonBurgerShape, BurgerShape, ButtonText }
  from 'components/ButtonBurgerShape/ButtonBurgerShape.component'

export { ButtonBurgerShape, BurgerShape, ButtonText }

export default ButtonBurgerShapeComponent
