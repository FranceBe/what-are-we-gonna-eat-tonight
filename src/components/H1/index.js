import H1Component, { H1, H1Default } from 'components/H1/H1.component'

export { H1, H1Default }

export default H1Component
