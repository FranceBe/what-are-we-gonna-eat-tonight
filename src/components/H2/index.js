import H2Component, { H2, H2Default } from 'components/H2/H2.component'

export { H2, H2Default }

export default H2Component
