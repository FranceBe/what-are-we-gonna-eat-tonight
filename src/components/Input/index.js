import InputComponent, { Input, InputField, Label, InputContainer } from 'components/Input/Input.component'

export { Input, InputField, Label, InputContainer }

export default InputComponent
