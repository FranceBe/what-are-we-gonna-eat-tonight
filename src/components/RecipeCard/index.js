import RecipeCardComponent, { RecipeCard } from 'components/RecipeCard/RecipeCard.component'

export { RecipeCard }

export default RecipeCardComponent
