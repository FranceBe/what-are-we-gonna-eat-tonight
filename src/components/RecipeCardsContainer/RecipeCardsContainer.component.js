import styled from 'styled-components'
import { spaces } from 'styles/variables'

export const RecipeCardsContainer = styled.div`
  display: flex;
  flex-direction: column;
  flex-wrap: wrap;
  justify-content: center;
  align-items: center;
  margin: ${spaces.regular} 0;
  & div:not(:last-of-type) {
    margin-bottom: ${spaces.medium};
  }
`
