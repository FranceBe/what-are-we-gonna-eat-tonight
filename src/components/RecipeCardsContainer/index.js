export { RecipeCardsContainer } from 'components/RecipeCardsContainer/RecipeCardsContainer.component'
