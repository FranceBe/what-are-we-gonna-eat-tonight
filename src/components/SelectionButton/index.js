import SelectionButtonComponent,
{ SelectionButton, SingleButton, ButtonsContainer }
  from 'components/SelectionButton/SelectionButton.component'

export { SelectionButton, SingleButton, ButtonsContainer }

export default SelectionButtonComponent
