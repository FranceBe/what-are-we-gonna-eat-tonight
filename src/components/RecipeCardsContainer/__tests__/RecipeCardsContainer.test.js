import React from 'react'
import renderer from 'react-test-renderer'
import { RecipeCardsContainer } from 'components/RecipeCardsContainer'

describe('RecipeCardsContainer', () => {
  it('should match snapshot', () => {
    const RecipeCardsContainerTree = renderer.create(<RecipeCardsContainer />).toJSON()
    expect(RecipeCardsContainerTree).toMatchSnapshot()
  })
})
