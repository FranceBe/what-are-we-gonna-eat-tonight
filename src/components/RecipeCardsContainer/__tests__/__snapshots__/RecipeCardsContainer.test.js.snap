// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecipeCardsContainer should match snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: 20px 0;
}

.c0 div:not(:last-of-type) {
  margin-bottom: 10px;
}

<div
  className="c0"
/>
`;
